app/controllers/games_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def create
     game = Game.create(white_player_id: current_user.id)
+    current_user << game
```

Comment:
Does this work?  Regardless, it might be better to make this more explicit:

```ruby
current_user.games << game
```

Or even:

```ruby
Participation.create(user: current_user, game: game)
```